src/AppBundle/Form/DataTransformer/ArrayToStringTransformer.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Form\DataTransformer;
+
+use AppBundle\Entity\Tag;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * This data transformer is used to translate the array of tags into a comma separated format
+ * that can be displayed and managed by Bootstrap-tagsinput js plugin (and back on submit).
+ *
+ * See http://symfony.com/doc/current/form/data_transformers.html
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class ArrayToStringTransformer implements DataTransformerInterface
```

Comment:
@yceruto,  I would name this class like `TagArrayToStringTransformer`, because the current name sounds very common.